includes/admin/onboarding/onboarding.php

```diff
@@ -584,13 +608,28 @@ public function ajax_onboarding_completed() {
 		}
 
 		// @todo - Add correct permissions check!
-
 		update_option( 'edd_onboarding_completed', current_time( 'Y-m-d H:i:s' ) );
 		update_option( 'edd_tracking_notice', true );
 
-		delete_option( 'edd_onboarding_started' );
-		delete_option( 'edd_onboarding_latest_step' );
+		$this->clean_onboarding_options();
+
+		exit;
+	}
+
+	/**
+	 * Ajax callback for skipping the Onboarding.
+	 *
+	 * @since 3.2
+	 */
+	public function ajax_onboarding_skipped() {
+		if ( ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'edd_onboarding_wizard' ) ) {
+			exit();
+		}
+
+		// @todo - Add correct permissions check!
+		update_option( 'edd_onboarding_completed', 0 );
```

Comment:
```suggestion
		update_option( 'edd_onboarding_completed', true );
```
Or I think this could still be a date/time even if skipped. But a truthy value feels safer for logical checks (like line 80) than setting it to a falsey value.